### ISSUE TITLE
Get my merge requests list

### ISSUE THREAD

**Issue by danakonovalenko** [edenlabllc/ehealth.api#4035]
As nhs admin reviewer I want to be able to get my merge requests in statuses ('POSTPONE', 'NEW')

In order to do that we need to implement WS based on [logic](https://edenlab.atlassian.net/wiki/x/nQJUKg) and schema

- [x] implement
- [ ] test

**Comment by ephor** [comment]
Blocked by edenlabllc/ehealth.api#4038

**Comment by danakonovalenko** [comment]
there is a problem with checking role, I still get merge requests list/id without required role `NHS REVIEWER` in case I have required scope

**Comment by svetavedmed** [comment]
- [ ] in case i do not have merge requests in status NEW i still receive 'canAssignNew: false'
![Screen Shot 2019-02-11 at 9.29.49 AM.png](https://zube.io/files/edenlabllc/f990b54157a7814425abca6107a967b9-screen-shot-2019-02-11-at-9-29-49-am.png)

**Comment by progsmile** [comment]
Now, max requests count is 5

**Comment by danakonovalenko** [comment]
in config `MAX_POSTPONED_MANUAL_MERGE_REQUESTS`= 5, however now it's I can have more. this parameters doesn't work

**Comment by progsmile** [comment]
This is not limit for query
It's particular count which should be checked on assign
